### ISSUE TITLE
clustermgr4 not starting in Ubuntu 18

### ISSUE THREAD

**Issue by mzico** [GluuFederation/cluster-mgr#294]
`clustermgr4-cli start` throwing below error: 

```
Starting Celery Beat
Traceback (most recent call last):
  File "/usr/local/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/celery/__main__.py", line 16, in main
    _main()
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 322, in main
    cmd.execute_from_commandline(argv)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 496, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 273, in execute_from_commandline
    argv = self.setup_app_from_commandline(argv)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 479, in setup_app_from_commandline
    self.app = self.find_app(app)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 501, in find_app
    return find_app(app, symbol_by_name=self.symbol_by_name)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/utils.py", line 359, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 504, in symbol_by_name
    return imports.symbol_by_name(name, imp=imp)
  File "/usr/local/lib/python2.7/dist-packages/kombu/utils/imports.py", line 56, in symbol_by_name
    module = imp(module_name, package=package, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/utils/imports.py", line 104, in import_from_cwd
    return imp(module, package=package)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/usr/local/bin/clusterapp.py", line 21, in <module>
    app = create_app()
  File "/usr/local/lib/python2.7/dist-packages/clustermgr/application.py", line 71, in create_app
    from clustermgr.views.index import index
  File "/usr/local/lib/python2.7/dist-packages/clustermgr/views/index.py", line 27, in <module>
    from clustermgr.core.ldap_functions import LdapOLC
  File "/usr/local/lib/python2.7/dist-packages/clustermgr/core/ldap_functions.py", line 7, in <module>
    from ldap3 import Server, Connection, SUBTREE, BASE, LEVEL, \
  File "/usr/local/lib/python2.7/dist-packages/ldap3/__init__.py", line 139, in <module>
    from .core.connection import Connection
  File "/usr/local/lib/python2.7/dist-packages/ldap3/core/connection.py", line 44, in <module>
    from ..operation.compare import compare_operation, compare_request_to_dict
  File "/usr/local/lib/python2.7/dist-packages/ldap3/operation/compare.py", line 28, in <module>
    from ..operation.search import ava_to_dict
  File "/usr/local/lib/python2.7/dist-packages/ldap3/operation/search.py", line 41, in <module>
    from pyasn1.error import PyAsn1UnicodeDecodeError
ImportError: cannot import name PyAsn1UnicodeDecodeError
Starting Gunicorn Web Server
```

Workaround: 

 - `pip install –upgrade pyasn1-modules` 
 - `clustermgr4-cli stop`
 - `clustermgr4-cli start`

**Comment by mzico** [comment]
CM version: 4.1.-3

**Comment by mbaser** [comment]
Fixed in 4.1.-4
